etl-pipeline/tests/unit/api/test_api_app.py

```diff
@@ -0,0 +1,30 @@
+import os
+import pytest
+
+from api.app import API
+
+
+class TestAPI:
```

Comment:
We could unit test the init, _register_blueprints, and create_error_responses but we start losing value here with the number of lines of codes required to mock everything.